Framework/Core/ANALYSIS.md

```diff
@@ -386,7 +386,19 @@ combinations(filter, tracks, covs); // equivalent to combinations(CombinationsFu
 
 ### Saving tables to file
 
-Produced tables can be saved to file as TTrees. This process is customized by the command line option `--keep` (of the internal-dpl-AOD-writer). **Please be aware, that the format of the `keep` option as described here is preliminary and might be changed in future.**
+Produced tables can be saved to file as TTrees. This process is customized by various command line options of the internal-dpl-AOD-writer. The options allow to specify which columns of which table are saved to which tree in which file.
+
+**Please be aware, that the functionality of these options is preliminary and might be changed in future.**
+
+The options to consider are:
+
+* --keep
+* --res-file
+* --ntfmerge
+* --json-file
```

Comment:
we definitely need to have more explicative names.

Reply:
I can think about better names. However, shouldn't we have common rules to create option names?

Reply:
I would avoid "half compressed" names. Notice that you should be able to specify a short version by providing a comma (maybe should be documented). Maybe for this particular case we should prefix them with "--aod" or something.